plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java

```diff
@@ -1049,4 +1052,25 @@ public static String sortingColumnToString(SortingColumn column)
     {
         return column.getColumnName() + ((column.getOrder() == DESCENDING) ? " DESC" : "");
     }
+
+    public static boolean filterSchema(String schemaName)
```

Comment:
i think i was writing a comment about this but cannot see it now... sorry for late thought then

this method is now public and no longer in context of use, we should give it a more descriptive name

`isHiveSystemSchema` (with inverse semantics)